code/modules/clothing/suits/armor.dm

```diff
@@ -316,3 +316,39 @@
 	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
 	armor = list("melee" = 25, "bullet" = 20, "laser" = 20, "energy" = 10, "bomb" = 20, "bio" = 50, "rad" = 20, "fire" = -10, "acid" = 50)
+
+//New and improved Tech Armor
+/obj/item/clothing/head/helmet/space/hardsuit/security_armor
+	name = "security exoskeleton helmet"
+	desc = "A specialized exoskeleton armor helmet built into a suit of armor; offers decent protection, and comes with a flash-resistant HUD visor and headlamp."
+	icon_state = "hardsuit0-secexo"
+	item_state = "hardsuit0-secexo"
+	hardsuit_type = "secexo"
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 55, "acid" = 70)
```

Comment:
Why is the helmet armour block different from the suit armour block?

Reply:
The stats are copy/pasted from the normal security helmets
![image](https://user-images.githubusercontent.com/57020074/90304607-96ca1a00-de87-11ea-951b-e0cf0ad40f93.png)